### PR TITLE
fix(refs DPLAN-1898): Destroy Tooltip instead of hiding

### PR DESCRIPTION
**Pull request by salisdemos**
It was the idea to hide the tooltip on mouseleave and only destroy it after 300ms to prevent recreating it again and again. But it turned out that it then sometimes got created twice. That had the effect, that the second tooltip didn't disappear.

**Ticket** DPLAN-1898

### CODE REVIEW

File: src/components/DpTooltip/utils/tooltip.js [hunk 1]
```diff
@@ -72,7 +63,7 @@ const initTooltip = (el, value, options) => {
     options,
     zIndex
   )
-  handleHideTooltip = () => hideTooltip(document.getElementById(el.getAttribute('aria-describedby')))
+  handleHideTooltip = () => deleteTooltip(document.getElementById(el.getAttribute('aria-describedby')))
```
Comment (spiess-demos) [inline src/components/DpTooltip/utils/tooltip.js:66]: Nitpick: As we're not actually hiding anything here, i'd suggest to unify wording:

```suggestion
  handleRemoveTooltip = () => removeTooltip(document.getElementById(el.getAttribute('aria-describedby')))
```

File: src/components/DpTooltip/utils/tooltip.js [hunk 2]
```diff
@@ -83,8 +74,6 @@ const initTooltip = (el, value, options) => {
 const showTooltip = async (id, wrapperEl, { place = 'top', container = 'body', classes = '' }, zIndex)  => {
   if (!document.getElementById(wrapperEl.getAttribute('aria-describedby'))) {
```
Comment (spiess-demos) [inline src/components/DpTooltip/utils/tooltip.js:75]: This function could be merged into `createTooltip`. This way, the wording would also be more appropriate (as we're not showing anything, just recreating the elements).

**Review by spiess-demos — COMMENTED** [review]
Nice! Only a few naming suggestions.

File: src/components/DpTooltip/utils/tooltip.js (hunk 2 above)
Comment (salisdemos) [reply src/components/DpTooltip/utils/tooltip.js:59]: like so?
843c1dee22976879111309c39c5ee7e470acbb14

File: src/components/DpTooltip/utils/tooltip.js (hunk 1 above)
Comment (salisdemos) [reply src/components/DpTooltip/utils/tooltip.js]: 843c1dee22976879111309c39c5ee7e470acbb14

**Review by spiess-demos — APPROVED** [review]
lgtm! (code review)

**Pull request merged**